inequality/_indices.py

```diff
@@ -11,6 +11,13 @@
      :cite:`nijkamp2015cultural`
      https://www.econstor.eu/bitstream/10419/107568/1/dp8782.pdf
 """
+import warnings
+
+warnings.warn(
+    "The module _indices is deprecated and will be removed on 2025-01-01.",
+    DeprecationWarning,
```

Comment:
DeprecationWarning assumes that we cut a release, then change it to FutureWarning and make another release with that, then enforce the deprecation. Shall we rather go straight for FutureWarning?

Reply:
Good point, let's change to a `FutureWarning`.